src/pages/UserSettings/UserSettings.tsx

```diff
@@ -0,0 +1,209 @@
+    }
+  }, [emailTime]);
+
+  // DANGEROUS delete account mutation
+  const deleteAccountMutation = useMutation({
+    mutationFn: () => api.delete("/user/"),
+    onSuccess: () => {
+      deleteUser();
+    },
+  });
+
+  // Updating email alerts setting
+  const updateEmailAlerts = useMutation({
+    // Only way to have optional parameters for mutationFn
+    // https://github.com/TanStack/query/issues/4264#issuecomment-1268054812
+    mutationFn: (enabled: boolean) =>
+      api
+        .post<SettingShort>("/user/settings/", {
+          key: "notify",
+          value: emailTime,
+          enabled: enabled,
+        })
+        .then((res) => res.data),
+    onSuccess: (data) => {
+      setHasEmailAlerts(data.enabled);
+      setEmailTime(data.value);
+      setEmailTimeNetworkingState(Constants.FINISHED_MUTATION_TEXT);
+    },
+  });
+
+  const { isLoading: usernameChangeLoading, mutate: changeUsername } =
+    useMutation({
+      mutationFn: (username: string) =>
+        api.put("/user/info/", {
+          data_type: "username",
+          data: username,
+        }),
+      onError: () => {
+        setChangeUsernameState("Username already taken.");
+      },
+      onSuccess: () => {
+        setChangeUsernameState("Username updated. Refresh to see changes.");
+      },
+    });
+
+  const { isLoading: emailChangeLoading, mutate: changeEmail } = useMutation({
+    mutationFn: (email: string) => {
+      return api.put("/user/info/", {
+        data_type: "email",
+        data: email,
+      });
+    },
+    onSuccess: () => {
+      setChangeEmailState("Email successfully updated");
+    },
+    onError: () => {
+      setChangeEmailState("Email may already be taken.");
+    },
+  });
+
+  return (
+    <>
+      <Column className="mx-8 my-16 w-full gap-6">
+        <h2>Settings</h2>
+        <h4 className="text-gray-600">Notification Settings</h4>
+        <Row className=" h-6 w-1/2 items-center">
+          <p className="text-gray-400">Email Alerts</p>
+          <div className="flex flex-1" />
+          <Toggle
+            toggled={hasEmailAlerts}
+            onToggle={() => {
+              updateEmailAlerts.mutate(!hasEmailAlerts);
+            }}
+          />
+        </Row>
+        {hasEmailAlerts && (
+          <>
+            <TextField
+              value={emailTime}
+              onChangeValue={(emailTime) => {
+                setEmailTimeNetworkingState(Constants.LOADING_MUTATION_TEXT);
+                setEmailTime(emailTime);
+              }}
+              label="Pick a time to receive notifications:"
+              type="time"
+            />
+            <label className="font-normal text-gray-600">
+              {emailTimeNetworkingState}
+            </label>
+          </>
+        )}
+        <h4 className="text-gray-600">Account Settings</h4>
+        <TextField
+          value={email}
+          hint="Enter a new email address"
+          onChangeValue={setEmail}
+          label="Change email:"
```

Comment:
super small thing but maybe make this just "Email"? Having "Change Email" twice looks/feels redundant